js/dashicons-picker.js

```diff
@@ -0,0 +1,422 @@
+			'screenoptions',
+			'cart',
+			'feedback',
+			'cloud',
+			'translation',
+			'tag',
+			'category',
+			'archive',
+			'tagcloud',
+			'text',
+			'media-archive',
+			'media-audio',
+			'media-code',
+			'media-default',
+			'media-document',
+			'media-interactive',
+			'media-spreadsheet',
+			'media-text',
+			'media-video',
+			'playlist-audio',
+			'playlist-video',
+			'controls-play',
+			'controls-pause',
+			'controls-forward',
+			'controls-skipforward',
+			'controls-back',
+			'controls-skipback',
+			'controls-repeat',
+			'controls-volumeon',
+			'controls-volumeoff',
+			'yes',
+			'no',
+			'no-alt',
+			'plus',
+			'plus-alt',
+			'plus-alt2',
+			'minus',
+			'dismiss',
+			'marker',
+			'star-filled',
+			'star-half',
+			'star-empty',
+			'flag',
+			'info',
+			'warning',
+			'share',
+			'share1',
+			'share-alt',
+			'share-alt2',
+			'twitter',
+			'rss',
+			'email',
+			'email-alt',
+			'facebook',
+			'facebook-alt',
+			'networking',
+			'googleplus',
+			'location',
+			'location-alt',
+			'camera',
+			'images-alt',
+			'images-alt2',
+			'video-alt',
+			'video-alt2',
+			'video-alt3',
+			'vault',
+			'shield',
+			'shield-alt',
+			'sos',
+			'search',
+			'slides',
+			'analytics',
+			'chart-pie',
+			'chart-bar',
+			'chart-line',
+			'chart-area',
+			'groups',
+			'businessman',
+			'id',
+			'id-alt',
+			'products',
+			'awards',
+			'forms',
+			'testimonial',
+			'portfolio',
+			'book',
+			'book-alt',
+			'download',
+			'upload',
+			'backup',
+			'clock',
+			'lightbulb',
+			'microphone',
+			'desktop',
+			'tablet',
+			'smartphone',
+			'phone',
+			'smiley',
+			'index-card',
+			'carrot',
+			'building',
+			'store',
+			'album',
+			'palmtree',
+			'tickets-alt',
+			'money',
+			'thumbs-up',
+			'thumbs-down',
+			'layout',
+			'align-pull-left',
+			'align-pull-right',
+			'block-default',
+			'cloud-saved',
+			'cloud-upload',
+			'columns',
+			'cover-image',
+			'embed-audio',
+			'embed-generic',
+			'embed-photo',
+			'embed-post',
+			'embed-video',
+			'exit',
+			'html',
+			'info-outline',
+			'insert-after',
+			'insert-before',
+			'insert',
+			'remove',
+			'shortcode',
+			'table-col-after',
+			'table-col-before',
+			'table-col-delete',
+			'table-row-after',
+			'table-row-before',
+			'table-row-delete',
+			'saved',
+			'amazon',
+			'google',
+			'linkedin',
+			'pinterest',
+			'podio',
+			'reddit',
+			'spotify',
+			'twitch',
+			'whatsapp',
+			'xing',
+			'youtube',
+			'database-add',
+			'database-export',
+			'database-import',
+			'database-remove',
+			'database-view',
+			'database',
+			'bell',
+			'airplane',
+			'car',
+			'calculator',
+			'ames',
+			'printer',
+			'beer',
+			'coffee',
+			'drumstick',
+			'food',
+			'bank',
+			'hourglass',
+			'money-alt',
+			'open-folder',
+			'pdf',
+			'pets',
+			'privacy',
+			'superhero',
+			'superhero-alt',
+			'edit-page',
+			'fullscreen-alt',
+			'fullscreen-exit-alt',
+			'',
+			'',
+			''
```

Comment:
Are these 3 just to maintain an even grid? or something else?

Reply:
Removed empty string from array.